### DISCUSSION TITLE
Recommended way of implementing Fontawesome into Vitepress

### DISCUSSION THREAD

**Discussion by MarkusKeck** [vuejs/vitepress#1713] [Q&A] [answered]
What is the recommended way of implementing Fontawesome into vitepress?

I could download the fontawesome files (css and webfont) and use it with plain html
```html
<i class="fa-solid fa-user"></i>
```

But there is also some documentation for adding it to vue 2 and 3
https://fontawesome.com/docs/web/use-with/vue/
https://fontawesome.com/docs/web/use-with/vue/add-icons

To be able to use the second option  guess I also have to edit the vite.config.ts?

Or is there already a better way using fontawesome with vitepress?

**Comment by brc-dd** [comment, marked as answer]
You won't need to add/edit the vite.config for this. Just register that component like this: https://vitepress.vuejs.org/guide/theme-introduction#registering-global-components

(Refer Vue 3.x docs on FA, it's showing v2 ones by default.)
